llvm/test/tools/llvm-symbolizer/output-style-json-code.test

```diff
@@ -12,12 +12,12 @@
 ## Expected a list with one empty object with default values.
 # RUN: llvm-symbolizer --output-style=JSON -e %p/Inputs/addr.exe 0x10000000 | \
 # RUN:   FileCheck %s --check-prefix=NOT-FOUND --strict-whitespace --match-full-lines --implicit-check-not={{.}}
-# NOT-FOUND:[{"Address":"0x10000000","ModuleName":"{{.*}}/Inputs/addr.exe","Symbol":[{"Column":0,"Discriminator":0,"FileName":"","FunctionName":"","Line":0,"StartAddress":"","StartFileName":"","StartLine":0}]}]
+# NOT-FOUND:[{"Address":"0x10000000","ModuleName":"{{.*}}/Inputs/addr.exe","Symbol":[{"Approximate":false,"Column":0,"Discriminator":0,"FileName":"","FunctionName":"","Line":0,"StartAddress":"","StartFileName":"","StartLine":0}]}]
```

Comment:
Not sure who created/maintains/has an interest in the json output - could you check version history and make sure someone who's interested in it is involved in the review. I'd wonder if we should omit the Approximate flag when it's false, to simplify clients that don't enable the feature/so they don't have to deal with new/surprising json.

Reply:
Removed the explicit entry of Approximate in json o/p by default. Added @spavloff to review the json o/p.